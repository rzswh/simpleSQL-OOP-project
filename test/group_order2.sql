CREATE DATABASE qAz;
CREATE DATABASE eDc;
CREATE DATABASE iJn;
CREATE DATABASE aCs;
SHOW DATABASES;
USE qAz;
CREATE TABLE qwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
CREATE TABLE sqwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 4);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 3);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 2);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 9);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 10);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 7);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 8);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 20);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 30);
INSERT INTO qwe(wer, ert, tyu) VALUES (1, 2, 25);
UPDATE qwe SET wer=5 WHERE tyu>9;
UPDATE qwe SET ert=10 WHERE tyu<15;
select * from qwe;
select rty, count(*) from qwe;
select count(*) from qwe order by rty;
select count(*) from qwe group by wer;
select ert, wer, count(rty) from qwe group by ert, wer order by ert;
select ert from qwe order by tyu;
DROP DATABASE aCs;
DROP DATABASE qAz;
DROP DATABASE eDc;
DROP DATABASE iJn;
