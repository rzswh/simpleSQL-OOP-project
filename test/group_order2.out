Database
aCs
eDc
iJn
qAz
wer	ert	rty	tyu
1	10	3	2
1	10	3	3
1	10	3	4
1	10	3	7
1	10	3	8
1	10	3	9
5	10	3	10
5	2	3	20
5	2	NULL	25
5	2	3	30
rty	COUNT(*)
NULL	1
3	9
COUNT(*)
1
9
COUNT(*)
6
4
ert	wer	COUNT(rty)
2	5	2
10	1	6
10	5	1
ert
10
10
10
10
10
10
10
2
2
2
